distribution/docs/src/main/jdocs/content/_configuring/securing-accounts-contents.adoc

```diff
@@ -0,0 +1,14 @@
+:title: Initial Startup
+:type: configuring
+:status: published
+:summary: System state at Initial startup.
+:sortOrder: 08
+
+===== Configuring Accounts
+
+* *{hardening-step}*
+
+${branding} does not define accounts or types of accounts to support access. ${branding} uses an attribute based access control (*ABAC*) model. For reference, ABAC systems control access by evaluating rules against the attributes of the entities (_subject_ and _object_), actions, and the environment relevant to a request.
```

Comment:
`attribute based access control (*ABAC*) model.`

I italicized the definition for clarity.